projects/ui/src/lib/components/po-image/samples/sample-po-image-basic/sample-po-image-basic.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'sample-po-image-basic',
+  templateUrl: './sample-po-image-basic.component.html'
+})
+export class SamplePoImageBasicComponent {
+  srcImage =
+    'https://st2.depositphotos.com/3580719/9494/v/450/depositphotos_94949500-stock-illustration-illustration-in-stained-glass-style.jpg';
```

Comment:
Sugiro trocar a imagem para a logo do PO UI para evitar problemas com possíveis direitos autorais da imagem usada.

```suggestion
  srcImage =
    'https://raw.githubusercontent.com/po-ui/po-angular/master/docs/assets/po-logos/po_color_bg.svg';
```

Reply:
ok, corrigido !